packages/network/src/network.js

```diff
@@ -1426,3 +1426,37 @@ export function prepareLoopbackProtocolHandler(zone, { watch, allVows }) {
 
   return makeLoopbackProtocolHandler;
 }
+
+/**
+ *
+ * @param {import('@agoric/base-zone').Zone} zone
+ * @param {ReturnType<import('@agoric/vow').prepareVowTools>} powers
+ */
+export const preparePortAllocator = (zone, { watch }) => {
+  const makePortAllocator = zone.exoClass(
```

Comment:
`makePortAllocator` doesn't seem to be used for anything here; maybe...

```suggestion
export const preparePortAllocator = (zone, { watch }) =>
  zone.exoClass(
```